<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[testCreateCustome]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="44024" started-at="2020-01-19T07:36:31Z" finished-at="2020-01-19T07:37:15Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="44024" started-at="2020-01-19T07:36:31Z" finished-at="2020-01-19T07:37:15Z">
      <class name="com.actitime.test.CustomerModule">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.actitime.test.CustomerModule@6833ce2c]" name="openBrowser" is-config="true" duration-ms="5308" started-at="2020-01-19T13:06:31Z" finished-at="2020-01-19T13:06:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="PASS" signature="loginApp()[pri:0, instance:com.actitime.test.CustomerModule@6833ce2c]" name="loginApp" is-config="true" duration-ms="8677" started-at="2020-01-19T13:06:36Z" finished-at="2020-01-19T13:06:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="FAIL" signature="testCreateCustome()[pri:0, instance:com.actitime.test.CustomerModule@6833ce2c]" name="testCreateCustome" duration-ms="22244" started-at="2020-01-19T13:06:45Z" finished-at="2020-01-19T13:07:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HDFC_001] but found [All Customers]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HDFC_001] but found [All Customers]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.actitime.test.CustomerModule.testCreateCustome(CustomerModule.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[testCreateCustome]]>
            </line>
          </reporter-output>
        </test-method> <!-- testCreateCustome -->
        <test-method status="PASS" signature="logoutApp()[pri:0, instance:com.actitime.test.CustomerModule@6833ce2c]" name="logoutApp" is-config="true" duration-ms="5269" started-at="2020-01-19T13:07:07Z" finished-at="2020-01-19T13:07:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutApp -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.actitime.test.CustomerModule@6833ce2c]" name="closeBrowser" is-config="true" duration-ms="2502" started-at="2020-01-19T13:07:12Z" finished-at="2020-01-19T13:07:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.actitime.test.CustomerModule -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
